tests/UI/campaigns/functional/BO/14_advancedParameters/05_import/02_importFile.js

```diff
@@ -0,0 +1,142 @@
+    page = await helper.newTab(browserContext);
+  });
+
+  after(async () => {
+    await helper.closeBrowserContext(browserContext);
+    await files.deleteFile(newPath);
+  });
+
+  it('should login in BO', async function () {
+    await loginCommon.loginBO(this, page);
+  });
+
+  it('should go to import page', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'goToImportPage', baseContext);
+
+    await dashboardPage.goToSubMenu(
+      page,
+      dashboardPage.advancedParametersLink,
+      dashboardPage.importLink,
+    );
+
+    await importPage.closeSfToolBar(page);
+
+    const pageTitle = await importPage.getPageTitle(page);
+    await expect(pageTitle).to.contains(importPage.pageTitle);
+  });
+
+  it(`should download ${sampleFile.args.type} sample file`, async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'downloadFile', baseContext);
+
+    sampleFilePath = await importPage.downloadSampleFile(page, sampleFile.args.type);
+
+    const doesFileExist = await files.doesFileExist(sampleFilePath);
+    await expect(doesFileExist, `${sampleFile.args.type} sample file was not downloaded`).to.be.true;
+  });
+
+  describe('Import file', async () => {
+    it(`should upload ${sampleFile.args.type} sample text file`, async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'importFile', baseContext);
+
+      await files.renameFile(sampleFilePath, newPath);
+
+      const uploadSuccessText = await importPage.uploadSampleFile(page, sampleFile.args.dropdownValue, newPath);
+      await expect(uploadSuccessText).contain(sampleFile.args.type);
+    });
+
+    it('should go to next import file step', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'nextStep', baseContext);
+
+      const panelTitle = await importPage.goToImportNextStep(page);
+      await expect(panelTitle).contain(importPanelTitle);
+    });
+
+    it('should start import file', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'confirmImport', baseContext);
+
+      const modalTitle = await importPage.startFileImport(page);
+      await expect(modalTitle).contain(importModalTitle);
+    });
+
+    it('should close import progress modal', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'closeImportModal', baseContext);
+
+      const fileTypeDropdownSelector = await importPage.closeImportModal(page);
+      await expect(fileTypeDropdownSelector).to.be.true;
+    });
+  });
+
+  describe('Check import success', async () => {
+    it('should go to customers page', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'goToCustomersPage', baseContext);
+
+      await dashboardPage.goToSubMenu(
+        page,
+        importPage.customersParentLink,
+        importPage.customersLink,
+      );
```

Comment:
Sorry I forget this one here 

```suggestion
      await importPage.goToSubMenu(
        page,
        importPage.customersParentLink,
        importPage.customersLink,
      );
```